### DISCUSSION TITLE
2024/01/22/unlock-system-version-restrictions-of-msix-packages/

### DISCUSSION THREAD

**Discussion by giscus** [hmsjy2017/blog-comment#4] [Announcements]
# 2024/01/22/unlock-system-version-restrictions-of-msix-packages/

前言微软商店很多软件软件对系统版本要求太高，甚至低几个小版本都安装不上。这很大程度上并不是因为兼容性问题，因此可以通过改包来解决。现有的教程大多是直接安装解压后的软件包，这样并不方便管理，因此本文采用了另一种方法。 下面以 Apple Music 为例谈谈具体方法。 下载 MSIX 安装包首先去 网页版微软商店 获取对应软件的链接，比如 Apple Music 的链接是这个： 1https://a

https://blog.iamsjy.com/2024/01/22/unlock-system-version-restrictions-of-msix-packages/

**Comment by yuanzui-cf** [comment]
我朝，heart beat

**Reply by Dev-HouKY-06** [reply]
没想到你也是被Apple Music卡住的，说明这个设计很不合理啊！

**Comment by Dev-HouKY-06** [comment]
没想到你也是被Apple Music卡住的，说明这个设计很不合理啊！
